src/client_side.h

```diff
@@ -383,6 +387,10 @@ class ConnStateData : public Server, public HttpControlMsgSink, private Independ
     /// whether PROXY protocol header is still expected
     bool needProxyProtocolHeader_;
 
+    /// the parsed PROXY protocol message or nil if there is
```

Comment:
Please remove `if there...` part of the sentence because it is dangerous imprecise (we may not have parsed the required message yet).